swagger/v1/component_schemas/ECFParticipantRetainedDeclaration.yml

```diff
@@ -1,50 +1,13 @@
-description: "An ECF retained participant declaration"
+description: "An participant retain declaration request body"
```

Comment:
It seems like we mix "participant" and "ECF participant" a lot in the docs. I think it is probably better to always explicitly state ECF or NPQ, what do you think?

Also minor, but I think this should be "retained" instead of "retain" as well?